rewrite-pattern.js

```diff
@@ -83,6 +84,43 @@ const getUnicodePropertyEscapeSet = (value, isNegative) => {
 	return set.clone();
 };
 
+const getUnicodePropertyOfStringsData = (property, isNegative) => {
+	const data = getCharacterClassEmptyData();
+
+	if (isNegative) {
+		throw new Error("Cannot negate unicode sequence property");
+	}
+
+	let rawData;
+	try {
+		rawData = require(`./data/sequence-properties/${ property }.js`);
```

Comment:
Now that these properties are defined at the Unicode level, we settled on "properties of strings" rather than "sequence property". Let's try to use that terminology as much as possible.